dapr-spring/dapr-spring-workflows/src/main/java/io/dapr/spring/workflows/config/DaprWorkflowsConfig.java

```diff
@@ -0,0 +1,57 @@
+package io.dapr.spring.workflows.config;
+
+import io.dapr.workflows.Workflow;
+import io.dapr.workflows.runtime.WorkflowActivity;
+import io.dapr.workflows.runtime.WorkflowRuntime;
+import io.dapr.workflows.runtime.WorkflowRuntimeBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.BeansException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ApplicationContextAware;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.ImportBeanDefinitionRegistrar;
+
+import java.util.Map;
+
+@Configuration
```

Comment:
I am not entirely sure why we need the `@ComponentScan` here. We just need the `@Configuration` to make sure that we register the workflows and activities.

Reply:
I would also rename the class to `DaprWorkflowsConfiguration`, just like other configuration classes.

Reply:
done